### ISSUE TITLE
Polymer's @apply mixins removed after minification

### ISSUE THREAD

**Issue by antonmoiseev** [jakubpawlowicz/clean-css#648]
Input file:
```css
.title {
  @apply(--paper-font-body1);
  flex: 1;
}
```
After I run `cleancss styles.css -o styles.min.css` the output is:
```css
.title{flex:1}
```

Is it possible to preserve `@apply` mixins?

**Comment by jakubpawlowicz** [comment]
Sure :+1:

**Comment by xiulunlin** [comment]
@jakubpawlowicz 
It seems this problem is not fixed when I download `3.4.0-pre`.
Also please support theme name in polymer's `:host` with a double-hyphen prefix. like:  
```css
:host {
    //theme name
    --live-head-theme: {
        line-height: 40px !important;
        vertical-align: middle;
        height: 40px;
        z-index: 999;
    }
}
```

**Comment by jakubpawlowicz** [comment]
@xiulunlin Thanks for testing. I'll give it another look. Could you please point me to a relevant Polymer CSS documentation re :host thing?

**Comment by xiulunlin** [comment]
Here is [polymer css minxins](https://www.polymer-project.org/1.0/docs/devguide/styling.html#custom-css-mixins).
Notice that the value of a `--live-head-theme` attribute could be a string `10px` or a block `{height:10px;}`.
And I find that in new version 
`a{@apply(--live-head-theme)}` 
works well but if a space is inserted it fails.
`a{ @apply(--live-head-theme)}`

**Comment by jakubpawlowicz** [comment]
:+1: thanks @xiulunlin - I'll make sure these mixins are supported in 3.4. Hopefully we won't need any special options.

**Comment by coreylenertz** [comment]
+1

**Comment by jakubpawlowicz** [comment]
Ok, this should be done now! See 21a9399ca86a5cac4eb6c1e8ce083e27aab0b107

**Comment by psxlover** [comment]
I was just about to open an issue about paper-spinner not working because @apply was removed.
Guess I was too late. Thanks for fixing it.

**Comment by jakubpawlowicz** [comment]
It's just been released with clean-css 3.4. :fireworks:
